topology/graph/traversal/traversal_parser.go

```diff
@@ -315,6 +317,11 @@ func (s *GremlinTraversalStepContext) Exec(last GraphTraversalStep) (_ GraphTrav
 			} else {
 				s.Params[0] = time.Unix(param, 0)
 			}
+		case *ForeverPredicate:
```

Comment:
I would prefer to keep it consistent, meaning having always the first argument setting the time and the second one(optional) setting the duration. Thus I think G.At(Now, Forever) only and not G.At(Forever)